src/datetime/mod.rs

```diff
@@ -36,10 +36,12 @@ use crate::{Datelike, Timelike, Weekday};
 use rkyv::{Archive, Deserialize, Serialize};
 
 #[cfg(feature = "rustc-serialize")]
+#[cfg_attr(docsrs, doc(cfg(feature = "rustc-serialize")))]
 pub(super) mod rustc_serialize;
 
 /// documented at re-export site
 #[cfg(feature = "serde")]
+#[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
```

Comment:
can be removed as this is documented at the top level instead

Reply:
Done.